core/unit_test/CMakeLists.txt

```diff
@@ -113,7 +113,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;OpenACC;HIP;SYCL)
     # file. That then exceeded the shell command line max length.
     set(${Tag}_SOURCES1A)
     foreach(Name
-        BitManipulationBuiltins
+        Abort
```

Comment:
This reordering in alphabetical order makes me a bit nervous.  Please submit it as a separate PR

Reply:
Yep, I've already run issue an issue before. I'll create a new one and let that merge first.